lib/static/new-ui/features/suites/components/SnapshotsPlayer/index.tsx

```diff
@@ -251,6 +269,48 @@ export function SnapshotsPlayer(): ReactNode {
         }
     }, []);
 
+    const onSpeedChange = useCallback((value: string[]): void => {
+        console.log(value);
+        if (value) {
```

Comment:
Here we can simplify code a little bit:
```
if (!value) {
  return;
}

// ...
```

Allows you to reduce the amount of nested code